src/Agent.Listener/JobDispatcher.cs

```diff
@@ -464,7 +464,10 @@ private async Task RunAsync(Pipelines.AgentJobRequestMessage message, WorkerDisp
 
                     // we get first jobrequest renew succeed and start the worker process with the job message.
                     // send notification to machine provisioner.
-                    await notification.JobStarted(message.JobId);
+                    var systemConnection = message.Resources.Endpoints.SingleOrDefault(x => string.Equals(x.Name, WellKnownServiceEndpointNames.SystemVssConnection, StringComparison.OrdinalIgnoreCase));
+                    var accessToken = systemConnection?.Authorization?.Parameters["AccessToken"];
+                    await notification.JobStarted(message.JobId, accessToken, systemConnection.Url);
```

Comment:
Just FYI, the systemconnection.url are different between a build and a release.
build: TFS url (xxx.visualstudio.com)
release: RM url (xxx.vsrm.visualstudio.com)

Reply:
make sure your monitor app handles this. :)